CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+# Contributing to Reservations
+Thanks so much for helping with the development of Reservations! This guide will cover the process for helping out with the project and should allow you to get started quickly and easily. If you need any help with contributing, please e-mail `stc-reservations{at}yale{dot}edu` to contact the core team.
+
```

Comment:
If you want we could also mention that they can @-mention the core team with @YaleSTC/reservations